code/modules/client/preference_setup/loadout/items/suit.dm

```diff
@@ -290,6 +290,7 @@
 	dominiacape["white dominian cape, caladius"] = /obj/item/clothing/accessory/poncho/dominia_cape/caladius/white
 	dominiacape["dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao
 	dominiacape["white dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao/white
+	dominiacape["tribunalist medical cape"] = /obj/item/clothing/accessory/poncho/dominia_cape/hospital
```

Comment:
This should probably be restricted to medical only. Same as how the "tribunalist medical beret" is currently.

Reply:
I don't really want to bloat the loadout buttons with a new cape

Reply:
I think it should be restricted, and consistent with other items.
Unfortunately, I think the bloat is already here, and the loadout will not get any less messy anytime soon.
(Tags would potentially help with the bloat, but... Yeah.)

Reply:
> I think it should be restricted, and consistent with other items. We should not be cutting corners, cause like, I think the bloat is already here, and the loadout will not get any less messy anytime soon. (Tags would potentially help with the bloat, but... Yeah.)

I just don't think this is a problem at all but ok